nilearn/plotting/surface/_plotly_backend.py

```diff
@@ -1,10 +1,28 @@
 import math
+from warnings import warn
 
+import numpy as np
+
+from nilearn import DEFAULT_DIVERGING_CMAP
+from nilearn._utils.helpers import is_kaleido_installed
+from nilearn.plotting.displays import PlotlySurfaceFigure
+from nilearn.plotting.js_plotting_utils import colorscale
 from nilearn.plotting.surface._backend import (
     VALID_HEMISPHERES,
+    BaseSurfaceBackend,
     _check_hemispheres,
+    _check_surf_map,
```

Comment:
I had missed that in the previous PR, but given that those functions are now being imported in another module, we should remove their leading underscore.

noticed because they were flagged in vs code as private functions unused in the module where they are defined

Reply:
I would suggest making this a small follow up PR

Reply:
In this PR I try not to touch such things. I just separate backend code as it is, and add code when necessary like `get_surface_backend` function that can easily be identified as change. 

What you suggest is in my list for the next PR to explicitly see what is changing from the original code.

Reply:
ok cool: just noticed it so wanted to flag it